tgb/linkproppred/dataset.py

```diff
@@ -275,7 +276,7 @@ def generate_processed_files(self) -> pd.DataFrame:
             OUT_NODE_TYPE = self.root + "/" + "ml_{}.pkl".format(self.name + "_nodeType")
 
         if (osp.exists(OUT_DF)) and (self.version_passed is True):
-            print("loading processed file")
+            vprint("loading processed file")

```

Comment:
consider logging where we actually read from otherwise its not a very helpful print
```suggestion
            vprint(f"loading processed file from {OUT_DF} and edge features from {OUT_EDGE_FEAT}")

```